src/components/ExceededCommentLength.js

```diff
@@ -1,27 +1,62 @@
-import React from 'react';
+import React, {PureComponent} from 'react';
 import PropTypes from 'prop-types';
+import {debounce} from 'lodash';
```

Comment:
Interesting, in the rest of the codebase I see that we use UnderscoreJS's `debounce` function, not the one from lodash. AFAICT they must be pretty much the same though.